babel.config.js

```diff
@@ -97,6 +98,11 @@ module.exports = function(api) {
           convertESM ? "@babel/transform-modules-commonjs" : null,
         ].filter(Boolean),
       },
+      {
+        test: "./packages/babel-polyfill",
+        presets: [["@babel/env", envOptsNoTargets]],
+        plugins: [["@babel/transform-modules-commonjs", null]],
```

Comment:
This could just be this ?
```suggestion
        plugins: ["@babel/transform-modules-commonjs"],
```

Reply:
I meant to use `false` here to disable the plugin, since it should be handled by browserify.

```suggestion
        plugins: [["@babel/transform-modules-commonjs", false]],
```